src/bin/jcli_app/transaction/mk_witness.rs

```diff
@@ -36,9 +37,14 @@ pub struct MkWitness {
     /// If omitted it will be printed to the standard output.
     pub output: Option<PathBuf>,
 
+    /// the type of witness to build: account, UTxO or Legacy UtxO
     #[structopt(long = "type", parse(try_from_str))]
     pub witness_type: WitnessType,
 
+    /// the hash of the block0, the first block of the blockchain
+    #[structopt(long = "block0-hash", parse(try_from_str))]
```

Comment:
There is a similar option `--genesis-block` in jormungandr which has been bothering me, because it can be a file path or a hash, but it's only interpreted as a path when it fails to parse as a hash.

Can we harmonize these options in both executables so that: 1) they use the same names; 2) clearly disambiguate between file and hash parameters?

Reply:
I am not going to change the parameters in jormungandr. But I agree, this was not a great choice in there. I'll rename it jcli for now.